doctrine/event_listeners_subscribers.rst

```diff
@@ -270,3 +270,55 @@ to the tag like so:
 
 .. _`The Event System`: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html
 .. _`the Doctrine Documentation`: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html#entity-listeners
+
+
+Priorities for Event Listeners
+--------------------------------
+
+In case you have multiple listeners for the same event you can control the order in which they are invoked using the `priority` attribute on the tag. Listeners with a higher priority are invoked first.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        services:
+            my.listener.with_high_priority:
```

Comment:
I actually followed the naming convention of the other examples in this file :wink: 

If you want I can rename all of them?

Reply:
Hmm, as this targets 2.8 why not, but then this would better be done in another PR. Let's wait for other maintainers opinion before making this change. Thanks!